include/OpenSpaceToolkit/Astrodynamics/Trajectory/Segment.hpp

```diff
@@ -53,6 +55,7 @@ class Segment
         Maneuver  ///< Maneuver
     };
 
+    /// @brief Once a segment is set up with an event condition, it can be solved, resulting in this segment's Solution.
```

Comment:
That's the case even if there is no `EventCondition` IIRC nah? 🤔 Adapt doc if that's the case

Reply:
Currently Segments can only be defined by an `EventCondition`